app/models/host/managed.rb

```diff
@@ -740,6 +740,10 @@ def pxe_loader
     explicit_pxe_loader || hostgroup.try(:pxe_loader)
   end
 
+  def pxe_loader_efi?
+    pxe_loader.include?('EFI')
```

Comment:
Is this related to this change? It seems useful, however it's not exposed in safemode, should it be added to the Host jail too?

Reply:
It is related to the IPv6 refactoring I've done. It vastly simplifies some if-statements. It needs to be added to safemode list, I forgot. Let me do that.